perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -289,6 +289,24 @@ sub map : Path('') : Args(2) {
     $c->res->body($image->{data});
 }
 
+sub lookup_by_ref : Private {
```

Comment:
As this is part of a front page search (and only sets stash stuff on success), perhaps better in Around.pm? Shrug.